cpp/models/ode_seir/model.h

```diff
@@ -108,8 +106,33 @@ class Model : public CompartmentalModel<InfectionState, Populations<InfectionSta
         return temp;
     }
 
+    /**
+    * @brief interpolate reproduction numbers at freely chosen time value that lies in between the time points of the given time series.
+    * values are linearly interpolated from their immediate neighbors from the old time points.
+    * @param y TimeSeries whose reproduction numbers we want to interpolate
+    * @param t_value double time at which we approximate the reproduction number
+    * @return interpolated reproduction number at given time value
+    */
+    IOResult<double> interpolate_reproduction_numbers(const mio::TimeSeries<ScalarType>& y, const double t_value)
```

Comment:
```suggestion
    IOResult<ScalarType> get_reproduction_number(ScalarType t_value, const mio::TimeSeries<ScalarType>& y) 
```

This is not well done over the whole software so far but please always use `ScalarType` for `double`